tests/unittest/_torch/multi_gpu/test_allreduce.py

```
@@ -697,3 +698,228 @@ def test_moe_finalize_allreduce_no_residual(mpi_pool_executor):
+def run_minimax_allreduce_rms_op(input: torch.Tensor, tensor_parallel_size: int,
+                                 tensor_parallel_rank: int,
+                                 rms_weights: torch.Tensor, eps: float):
+    torch.manual_seed(42)
+
+    total_tokens = input.shape[0]
+    origin_dtype = input.dtype
+
+    input = input.cuda()
+    rms_weights = rms_weights.cuda()
+
+    input = input.reshape(total_tokens, tensor_parallel_size,
```

Ruff (0.15.9)

[error] Variable `input` is shadowing a Python builtin

(A001)